files/image_config/platform/linux_kernel_bde.conf

```diff
@@ -0,0 +1 @@
+blacklist linux_kernel_bde
```

Comment:
who is trying to load linux_kernel_bde if we are not running sonic service? 
backlist does not prevent it from being loaded as a dependency for another kernel module that is not blacklisted, are there any other modules depending on this module that could still start this?

Reply:
it is automatically loaded by kernel based on the pcie id. @jipanyang , user-bde and bnet bde are not pci driver, they will not be automatically loaded.